### ISSUE TITLE
Failed to fetch local file in windows

### ISSUE THREAD

**Issue by YF1999** [hyperjump-io/json-schema-core#6]
`FetchErrro` will be raised if trying to get a local file in windows:

```ts
// javascript or typescript code
const schema = await JsonSchema.get('file:///C:/path/to/my/schema/schema.json');

// error
// (node:23068) UnhandledPromiseRejectionWarning: FetchError: Invalid response body while trying to fetch : ENOENT: no such file or directory, open 'C:\C:\path\to\my\schema\schema.json'
//     at ReadStream.<anonymous> (D:\Projects\YF1999\easy-projs\node_modules\node-fetch\lib\index.js:399:12)
//     at ReadStream.emit (events.js:387:35)
//     at internal/fs/streams.js:126:14
//     at FSReqCallback.oncomplete (fs.js:180:23)
```

The reason is that, in [fetch.js](https://github.com/hyperjump-io/json-schema-core/blob/master/lib/fetch.js) the path converted from that url is `/C:/path/to/my/schema/schema.json`.

I don't know much about URL, but it should be `C:/path/to/my/schema/schema.json`.

**Comment by jdesrosiers** [comment]
Thanks for reporting this! It should be an easy fix.

**Comment by jdesrosiers** [comment]
Fix released as v0.21.1
